modules/oc-mirror-image-set-config-examples.adoc

```diff
@@ -5,16 +5,22 @@
 
 :_mod-docs-content-type: REFERENCE
 [id="oc-mirror-image-set-examples_{context}"]
-= Image set configuration examples
+= Tuning the Image set configuration
 
 The following `ImageSetConfiguration` file examples show the configuration for various mirroring use cases.
 
-// Moved to first; unchanged
-[discrete]
-[id="oc-mirror-image-set-examples-shortest-upgrade-path_{context}"]
-== Use case: Including the shortest {product-title} update path
+[id="oc-mirror-image-set-example-specific-ocp-release_{context}"]
+== Use case: Preparing for installing a specific {product-title} release
+
+It is recommended to pin the platform version to be close to what you are expecting to deploy. The simplest method is to set, on the channel level, the minVersion you expect to deploy. It may make sense to set minVersion to the latest version in the default channel, and leave it at that.
+
+Consider not setting maxVersion at all. In fact, if you have set the minVersion to the latest possible version, then no additional versions would be mirrored anyways.
 
-The following `ImageSetConfiguration` file uses a local storage backend and includes all {product-title} versions along the shortest update path from the minimum version of `4.11.37` to the maximum version of `4.12.15`.
+Also, on every invocation of oc-mirror with this image set configuration, the latest release of the stable-4.15 channel is evaluated. So running oc-mirror at regular intervals ensures that you automatically receive the latest releases of OpenShift Container Platform images.
```

Comment:
Not sure if this `stable-4.15` would be true when 4.16 releases.

Reply:
Can you confirm this once?